transport/mqtt/pom.xml

```diff
@@ -46,5 +46,16 @@
             <groupId>org.eclipse.paho</groupId>
             <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.kapua</groupId>
+            <artifactId>kapua-qa-markers</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/commons-logging/commons-logging -->
+        <dependency>
```

Comment:
Why this dependency has been added?

Reply:
Hi, @Coduz, Without this dependency, I had an error for some tests `java.lang.NoClassDefFoundError: org/apache/commons/logging/Log `after adding this dependency everything worked fine.

Reply:
That Exception means that a `slf4j-api` implementation is missing. So adding `commons-logging` will fix the issue, but in the "wrong" way in the sense that we use `logback` as implementation of `slf4j` . 
So this dependency should be changed to `logback`  with `<scope>test</scope>`